back/engines/commercial/bulk_import_ideas/app/views/bulk_import_ideas/web_api/v1/export_form.html.erb

```diff
@@ -14,51 +14,63 @@
   <% end %>
 
   <style>
-      @page {
-        size: A4;
-        margin: 40px 40px 80px;
-        @bottom-right {
-          font-family: <%== font_family %>;
-          content: '<%= page_copy %> ' counter(page);
+    @page {
+      size: A4;
+      margin: 40px 40px 80px;
+      @bottom-right {
+        font-family: <%== font_family %>;
+        content: '<%= page_copy %> ' counter(page);
+        margin-bottom: 10px;
+      }
+    }
+
+    body { font-family: <%== font_family %>; }
+    h1 { font-size: 18pt; }
+    h2 { font-size: 16pt; }
+    h3 { font-size: 14pt; }
+    h4, p, li { font-size: 12pt; line-height: 1.2; }
+
+    div.line { border-bottom: solid 1px #000; margin: 10px 0; height: 25px; }
+    div.select, div.ranking { display: flex; align-items: center; margin-top: -6px; }
+    div.checkbox { border: solid 1px #000; width: 12px; height: 12px; margin: 8px 10px 8px 2px; display: inline-block; }
+    div.ranking-box { border: solid 1px #000; width: 24px; height: 24px; margin: 8px 10px 8px 2px; display: inline-block; }
+    div.radio { border-radius: 50%; }
+    span.optional { font-weight: normal; }
+    div.field-group { page-break-inside: avoid; }
+
+    #header {
+      margin-bottom: 30px;
+      img { height: auto; }
+      h1 { margin-top: 0; }
+    }
+
+    #personal-data {
+      page-break-after: always;
+    }
+
+    #questions {
+      h2, h3 { margin-bottom: 8px; }
+      span.number { width: 30px; display: inline-block; }
+      div.content { margin-left: 30px; margin-right: 30px;}
+      p { margin-top: 0; margin-bottom: 4px;}
+      div.question, div.page { margin-bottom: 30px; }
+      div.unsupported p, div.visibility_disclaimer p { color: #666; font-style: italic; }
+      div.image-select {
+        display: grid;
+        grid-template-columns: repeat(3, 1fr);
+        gap: 10px;
+        margin-bottom: 10px;
+        div.image {
```

Comment:
Did we test this with broader/landscape mode images?

Reply:
Aha, you've found a bug! Will fix this.